feature-about/src/main/java/io/github/droidkaigi/confsched2022/feature/about/About.kt

```diff
@@ -53,15 +49,17 @@ fun AboutScreenRoot(
     modifier: Modifier = Modifier,
     showNavigationIcon: Boolean = true,
     onNavigationIconClick: () -> Unit = {},
+    onLinkClick: (url: String, packageName: String?) -> Unit = { _, _ -> },
```

Comment:
I think we can make a sealed interface or class for this. It makes the app more flexible.
But I will make a issue for it 👍

Reply:
You're right!
Actually I want to know opinions about this.
Now someone will follow this to another issue. 🤗